android/app/build.gradle

```diff
@@ -23,7 +23,7 @@ if (flutterVersionName == null) {
 }
 
 android {
-    namespace "com.example.fluttercon_2024"
```

Comment:
Is there a reason for changing the app ID (I can understand the change from example to android254 but not flutterocon_2024 to fluttercon2024)?
cc @sam-baraka Would this affect the CI/CD pipeline you had created?

Reply:
let me revert to fluttercon_2024, my bad